tests/hash/test_parse_checksum.py

```diff
@@ -0,0 +1,168 @@
+               'platform': 'linux64',
+               'version': '53.0b9'},
+      'url': 'https://archive.mozilla.org/pub/firefox/candidates/53.0b9-candidates/build1/SHA512SUMS',
+      'hash': 'a71d5b34a58b7c440711f838e268a17a7a515d6f641c1c29e1fde5e5313aab73' \
+              '2dfc38970e40f3cd209bbb32ad1618358fa5ebba77b0e6d88c2a44c46f630cc0'},
+       # -p linux64 -v latest
+     {'args': {'application': 'firefox',
+               'platform': 'mac',
+               'version': '53.0b9'},
+      'url': 'https://archive.mozilla.org/pub/firefox/candidates/53.0b9-candidates/build1/SHA512SUMS',
+      'hash': 'ec13593e7ba46c4f86ab15b888c33346a05efe0144120f8ac7d65633b5d7e259' \
+              '6e317bf073647cf04db400ae0f5d97411a24bf1f0a9691f49ca51ac995f339d6'},
+]
+
+tests_daily_scraper = [
+    # -p win32 -v latest
+    {'args': {'application': 'firefox',
+              'platform': 'win32',
+              'version': 'latest'},
+     'url': 'https://archive.mozilla.org/pub/firefox/candidates/52.0.2-candidates/build1/SHA256SUMS',
+     'hash': '197c5d03735b3c1084abee7a19aa23a2ec705cf5ee2e7accf88e0669cc99427a'},
+     # -p win64 -v latest
+     {'args': {'application': 'firefox',
+               'platform': 'win64',
+               'version': 'latest'},
+      'url': 'https://archive.mozilla.org/pub/firefox/candidates/52.0.2-candidates/build1/SHA256SUMS',
+      'hash': '4662484c180bad22e4077c14553f98dd8456b2b169f82299ed24de2b92cdd3de'},
+      # -p linux -v latest
+     {'args': {'application': 'firefox',
+               'platform': 'linux',
+               'version': 'latest'},
+      'url': 'https://archive.mozilla.org/pub/firefox/candidates/52.0.2-candidates/build1/SHA256SUMS',
+      'hash': '489a029538f8980458e49d7128d4791443038a6f150cfee472b59c7d04867f12'},
+       # -p linux64 -v latest
+     {'args': {'application': 'firefox',
+               'platform': 'linux64',
+               'version': 'latest'},
+      'url': 'https://archive.mozilla.org/pub/firefox/candidates/52.0.2-candidates/build1/SHA256SUMS',
+      'hash': 'a37f817182da13dd34520039da8d0ddeb8bad7dd260ec8fb79c61e770bbaf83f'},
+       # -p linux64 -v latest
+     {'args': {'application': 'firefox',
+               'platform': 'mac',
+               'version': 'latest'},
+      'url': 'https://archive.mozilla.org/pub/firefox/candidates/52.0.2-candidates/build1/SHA256SUMS',
+      'hash': 'f9a91e0d57652566ba7f0d2e191cdc3db3219b3cf29a83ed145a9110a0bda108'},
+]
+
+
+class FirefoxHashTests(unittest.TestCase):
+    """Test all scraper classes for Firefox against the remote server"""
+
+    def setUp(self):
+        logging.basicConfig(format=' %(levelname)s | %(message)s', level=logging.ERROR)
+        self.logger = logging.getLogger(self.__class__.__name__)
+
+        # Create a temporary directory for potential downloads
+        self.temp_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        mozfile.rmtree(self.temp_dir)
+
+    def test_release_scraper(self):
```

Comment:
I would propose to put the tests separated by scraper class into the appropriate sub folder under `tests`. So that they are located closely to the other tests of the same scraper type.